pages/views/_layouts/basic.njk

```diff
@@ -3,15 +3,16 @@
 
 {% block head %}
   <script>
-    /* Polyfill */
     (function () {
-      let version = '';
-      if (!window.Promise || !window.CustomEvent) {
-        version = 'full';
-      } else if (!('getRootNode' in Element.prototype) || !window.customElements || !window.IntersectionObserver || !window.ResizeObserver) {
-        version = 'light';
-      }
-      (version) && document.write('<script type="text/javascript" src="{{ '/bundles/polyfill-' | url }}' + version + '.js"><\/script>');
+       let version = '';
```

Comment:
Should use `var` instead of `let` as we support browsers that don't support es6.